edx_exams/apps/core/tests/test_api.py

```diff
@@ -882,3 +890,106 @@ def test_not_passed_due_no_due_date(self):
             exam = {'due_date': None}
 
             self.assertFalse(is_exam_passed_due(exam))
+
+
+class TestGetEscalationEmail(ExamsAPITestCase):
+    """
+    Tests for the get_escalation_email API function.
+    """
+    def setUp(self):
+        super().setUp()
+
+        self.exam = ExamFactory(
+            provider=self.test_provider,
+        )
+
+    def test_get_escalation_email(self):
+        """
+        Test that the escalation is returned correctly when a course is configured to use exams.
+        """
+        expected_escalation_email = 'test@example.com'
+
+        CourseExamConfigurationFactory(
+            course_id=self.exam.course_id,
+            provider=self.test_provider,
+            escalation_email=expected_escalation_email,
+        )
+
+        escalation_email = get_escalation_email(self.exam.id)
+
+        self.assertEqual(escalation_email, expected_escalation_email)
+
+    def test_get_escalation_email_no_configuration(self):
+        """
+        Test that None is returned correctly when a course is not configured to use an exam.
+        """
+        self.assertEqual(get_escalation_email(self.exam.id), None)
+
+
+class TestCreateOrUpdateCourseExamConfiguration(ExamsAPITestCase):
+    """
+    Tests for the create_or_update_course_exam_configuration API function.
+
+    This API function is primarily a wrapper around the CourseExamConfiguration.create_or_update method, so the
```

Comment:
I'd end up duplicating a lot of test logic here from the `test_models.py` file if I were to fully cover the behavior of the API function. I could go either way on that point, so let me know if you have a strong opinion.

Reply:
the duplication feels like it would be overkill so this lgtm.